fresque/lib/views.py

```diff
@@ -73,10 +78,17 @@ def newpackage(db, method, data, username):
             db.commit()
         except SQLAlchemyError:
             result.flash.append(("An error occurred while adding your "
-                 "package, please contact an administrator.", "danger"))
+                "package, please contact an administrator.", "danger"))
         else:
             result.flash.append(("Package successfully created!", "success"))
             result.redirect = ('package', {"name": pkg.name})
+
+            # Add git repo creation message
+            if create_git_repo(pkg.name, gitfolder):
```

Comment:
Yeah, the IOError is not handled, in case of error the user will just see an error 500 page. It would be better to catch the exception and handle it.
Plus, the "if" construct here is strange, the create_git_repo function does not return True or False. It's a very C-like way of writing code ;-) Better use try/except/else.